components/odh-notebook-controller/controllers/suite_test.go

```diff
@@ -79,6 +88,37 @@ const (
 	odhNotebookControllerTestNamespace = "redhat-ods-applications"
 )
 
+// getOAuthProxyImageFromParams reads the oauth-proxy-image value from config/base/params.env
+// Returns empty string on error to allow graceful handling by caller
+func getOAuthProxyImageFromParams() string {
```

Comment:
I'd exit the binary with error, in this function, if getting oauth proxy fails... Just suggestion